src/posix.mak

```diff
@@ -142,9 +142,29 @@ all: dmd
 dmd: $G/dmd $G/dmd.conf
 .PHONY: dmd
 
+ifeq (osx,$(OS))
+
+CURL_FLAGS := -fsSL --retry 5 --retry-max-time 120 --connect-timeout 5 --speed-time 30 --speed-limit 1024
+LDC_VER := 1.19.0-beta2
+LDC_ZIP := ldc2-$(LDC_VER)-osx-x86_64
+LDC_URL := https://github.com/ldc-developers/ldc/releases/download/v$(LDC_VER)/$(LDC_ZIP).tar.xz
+LDC_DIR := $(GENERATED)/$(LDC_ZIP)
+LDC := $(LDC_DIR)/bin/ldc2
+
+$(LDC): $(GENERATED)
```

Comment:
I think generated might get cleaned during the build.  If so, maybe it would be better to put this somewhere else?  Otherwise it might get downloaded multiple times during the build?  We should check the logs to make sure.

Reply:
Moved it to the root folder.